### ISSUE TITLE
A11y_SQLAzureDataStudio_Notebook_Create new notebook_ScreenReader: The change in button is not announced.

### ISSUE THREAD

**Issue by v-atkulk** [microsoft/azuredatastudio#9406]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11y_SQLAzureDataStudioBenchmark; #A11yMAS; #A11ySev2; #A11yTCS; #SQL Azure Data Studio; #Benchmark; #DesktopApp; #VoiceOver; #MacOs; #MAS1.3.1; #MAS4.2.12;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.15.0-insider
Commit: 3a8b74a
Date: 2020-02-11T05:49:41.181Z
VS Code: 1.42.0
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Darwin x64 19.3.0
Mac OS:10.15.3

**Additional Details:**
Mac OS:10.15.3
MAS References:MAS1.3.1, MAS4.2.12
Screen reader: VoiceOver

**Repro Steps:**
1.Launch Azure Data Studio.
2.Click File then New Notebook.
3.Launch screen reader as VoiceOver.
4.Use tab and navigate to the buttons "Trusted cell"

**Actual:**
On navigating to the button "Trusted cell" using tab press enter it changes to "Not trusted" but there is no announcement of the same.
Similar for "Collapse cell" button

**Expected:**
The change in name of the button should be announced.

**User Impact:**
Screen reader users will not know the current value of the button.

**Attachment link for Reference :**
[9406_A11y_SQLAzureDataStudio_Notebook_Create new notebook_ScreenReader.zip](https://github.com/microsoft/azuredatastudio/files/4297952/9406_A11y_SQLAzureDataStudio_Notebook_Create.new.notebook_ScreenReader.zip)

**Comment by alanrenmsft** [comment]
adding the attribute aria-live="polite" should fix the issue

**Comment by Charles-Gagnon** [comment]
Discussed this in office hours with v-tayazd - for actions which execute immediately such as these we shouldn't be announcing that an action is occurring as that tends to be too verbose.

**Comment by Ashutoshsahoo164** [comment]
As per the above mentioned comments closing this bug.

GitHubTags:#Closed;

**Comment by mstechie** [comment]
GitHubTags:#BM-TCS-SQLADS-MAC-Feb2020;
